### ISSUE TITLE
mass数据结果

### ISSUE THREAD

**Issue by Coleddie** [Henryjiepanli/Uncertainty-aware-Network#12]
您论文里提到：For the Massachusetts building dataset, we set the initial learning rate to 5e−4 and the batch size to 2，但是所给代码里lr = 1e-4，所以我用5e−4跑的实验，跑出结果F1:80.92，mIOU:67.95，Pre:87.37，Recall:75.36，在buildformer框架上复现的，其余两个数据跑出来相差没这么明显，算正常。请问是什么原因？

**Comment by Henryjiepanli** [comment]
这个不太正常，太低了，我看了一下，马赛诸州学习率确实是5e-4，确认一下其他环节有没有问题。

-----原始邮件-----
发件人:Cola ***@***.***>
发送时间:2024-12-10 17:06:48 (星期二)
收件人: Henryjiepanli/Uncertainty-aware-Network ***@***.***>
抄送: Subscribed ***@***.***>
主题: [Henryjiepanli/Uncertainty-aware-Network] mass数据结果 (Issue #12)






您论文里提到：For the Massachusetts building dataset, we set the initial learning rate to 5e−4 and the batch size to 2，但是所给代码里lr = 1e-4，所以我用5e−4跑的实验，跑出结果F1:80.92，mIOU:67.95，Pre:87.37，Recall:75.36，在buildformer框架上复现的，其余两个数据跑出来相差没这么明显，算正常。请问是什么原因？

**Comment by Baccarali** [comment]
我曾经也复现过Buildformer的代码，在WHU数据集上与原论文报告的结果差距不是很大，在Massachusetts数据集上复现的结果(IoU)远远达不到论文报告的结果

**Comment by Henryjiepanli** [comment]
不太对吧，这个马塞诸州蛮好跑到的

----- Original Message -----

**Comment by Baccarali** [comment]
Buildformer原论文中描述的马塞诸州的是用加载WHU的权重在其基础上进行训练（For the Massachusetts building dataset and the Inria Aerial Image Labeling dataset, we used BuildFormer’s weight trained on the WHU building dataset and then fine-tuned it
for 105 epochs with a learning rate of 5e − 4），会不会是这个原因，我是在没有加任何预训练权重的基础上进行的实验，最终结果差了很多。

**Comment by Henryjiepanli** [comment]
这个没什么影响，你这个70不到的iou肯定是有问题的，检查一下数据什么的有没有问题。


-----原始邮件-----
发件人:Baccarali ***@***.***>
发送时间:2024-12-11 16:23:49 (星期三)
收件人: Henryjiepanli/Uncertainty-aware-Network ***@***.***>
抄送: Li ***@***.***>, Comment ***@***.***>
主题: Re: [Henryjiepanli/Uncertainty-aware-Network] mass数据结果 (Issue #12)






Buildformer原论文中描述的马塞诸州的是用加载WHU的权重在其基础上进行训练（For the Massachusetts building dataset and the Inria Aerial Image Labeling dataset, we used BuildFormer’s weight trained on the WHU building dataset and then fine-tuned it
for 105 epochs with a learning rate of 5e − 4），会不会是这个原因，我是在没有加任何预训练权重的基础上进行的实验，最终结果差了很多。

**Comment by Baccarali** [comment]
好的，非常感谢您的回复，我的一篇论文审稿专家要求增加Inria数据集，为了方便和其他模型参数进行对比，想问一下作者关于这个数据集的训练过程也是加载了WHU的权重进行微调还是未加预训练权重重新训练的

**Comment by Henryjiepanli** [comment]
好的，感谢，我没有用预训练

----- Original Message -----

**Comment by Baccarali** [comment]
谢谢！

**Comment by Coleddie** [comment]
好的感谢，请问①mass的数据集是512格式对结果有影响吗，我看SDSC-UNet和DSAT-Net是这样做的 ②另外三个数据集都是512的基础上buidlformer能复现差不多的结果，说明是通的，UAnet模型移过去，whu和inria指标都差不多，唯独mass出现问题 ③我的实验都是在whu训练完的基础上，然后放到其他两个上进行微调的，因为大家论文里也都是这么阐述的。

**Comment by Henryjiepanli** [comment]
512也行，你把dataloader用whubuilding或者inria的就行，其他超参和whubuilding保持一致就ok

----- Original Message -----

**Comment by sultonouv** [comment]
Hello. I have two questions:

1. As it is well-known, the Inria dataset has no test labels. How did you evaluate your model on Inria dataset? Did you report the results on the validation set or did you use the official scoring tool by the dataset owners? 

2. Another question is that Buildformer paper trains their model from scratch on WHU dataset and fine tunes on the other two datasets? Did you apply the same approach or are all models trained from scratch on all three datasets? 

Thank you!

**Comment by Henryjiepanli** [comment]
Thank you for your attention to our work. Regarding the evaluation on Inria dataset, we report the results on the validation set, which is described as "Referring to the official suggestion, we select one to five tiles from each city for validation and the rest for training" in the paper. For the second question, we didn't apply the finetuning strategy, we directly trained the model on each dataset.

-----原始邮件-----
发件人:"Furkat Sultonov" ***@***.***>
发送时间:2025-01-04 13:46:42 (星期六)
收件人: Henryjiepanli/Uncertainty-aware-Network ***@***.***>
抄送: Li ***@***.***>, Comment ***@***.***>
主题: Re: [Henryjiepanli/Uncertainty-aware-Network] mass数据结果 (Issue #12)






Hello. I have two questions:

As it is well-known, the Inria dataset has no test labels. How did you evaluate your model on Inria dataset? Did you report the results on the validation set or did you use the official scoring tool by the dataset owners?

Another question is that Buildformer paper trains their model from scratch on WHU dataset and fine tunes on the other two datasets? Did you apply the same approach or are all models trained from scratch on all three datasets?

Thank you!
